### PR TITLE
Removed usage of lombok

### PR DESCRIPTION
**Pull request by jbee**
This PR removes the dependency from the project and includes the changes do in PRs #3642 #3641 #3640 #3639 #3638.

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/web/gf-web-connector/src/main/java/fish/payara/appserver/context/JavaEEContextUtilImpl.java [hunk 1]
```diff
@@ -85,6 +82,15 @@ void init() {
         doSetInstanceContext();
     }
 
+    public ServerContext getServerContext() {
```
Comment (MattGill98) [inline appserver/web/gf-web-connector/src/main/java/fish/payara/appserver/context/JavaEEContextUtilImpl.java]: Just to make sure we make no changes, this should be package private.

File: appserver/ejb/ejb-container/pom.xml [hunk 2]
```diff
@@ -276,11 +276,5 @@
             <artifactId>payara-api</artifactId>
```
Comment (MattGill98) [inline appserver/ejb/ejb-container/pom.xml]: Missing copyright headers in most of these poms!

File: appserver/web/gf-web-connector/src/main/java/fish/payara/appserver/context/JavaEEContextUtilImpl.java (hunk 1 above)
Comment (jbee) [reply appserver/web/gf-web-connector/src/main/java/fish/payara/appserver/context/JavaEEContextUtilImpl.java]: FYI: This was not a miss. When the changes were still in separate branches making it this method `protected` would result in an compile error in my IDE for some odd reason. That is why I left it `public`. Now with everything in one branch it behaves as expected and lets me change it without complaining.

File: appserver/ejb/ejb-container/pom.xml (hunk 2 above)
Comment (jbee) [reply appserver/ejb/ejb-container/pom.xml]: I updated the pom headers now as well.

**Comment by jbee** [comment]
Oh I didn't notice there were different ones. Can you please be clear about what we want to have as outcome? Replace Oracle headers? Add Payara on top? Modify just those that have a Payara header?

**Comment by Cousjava** [comment]
All of the poms already have a Payara copyright, just extend the date for them. Note that the Payara copyright is at the bottom of the copyright header, not next to the Oracle one.

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by jbee** [comment]
Jenkins test please

**Comment by MattGill98** [comment]
Jenkins test please

**Comment by jbee** [comment]
Jenkins test please

**Comment by jbee** [comment]
Jenkins test please

File: appserver/common/container-common/src/main/java/com/sun/enterprise/container/common/impl/util/ClusteredSingletonLookupImplBase.java [hunk 3]
```diff
@@ -60,32 +63,35 @@
...
 
-    protected String getMapKey() {
-        return mapKey;
+    public final String getSessionHzKey() {
+        return sessionHzKey.updateAndGet(v -> v != null ? v : makeSessionHzKey());
     }
 
-    public String getSessionHzKey() {
-        return sessionHzKey;
+    protected final void update() {
```
[the remark this answers is not in the document]
Comment (jbee) [reply appserver/common/container-common/src/main/java/com/sun/enterprise/container/common/impl/util/ClusteredSingletonLookupImplBase.java]: I am glad you asked :) I were describing this in _Core Server_ channel. With lombok's `@Getter` the fields were lazy initialised to a final state. That means that on first call of any of the getters the according `makeXyZ` method was computing whatever became the final value of the field. With `makeSessionHzKey` and `makeLockKey` being dependent on `getClusteredSessionKey` which reads a field that can be changed via `ClusteredSingletonLookupImpl.setClusteredSessionKey` it looks like the result of the getter should actually update when `setClusteredSessionKey` is called to change the basis on which `getClusteredSessionKey` is computed. Usage of `setClusteredSessionKey` suggests this is _dynamic_. While this is a change of behaviour to the lombok `@Getter` version I think it is what we were trying to make happen. So when `setClusteredSessionKey` is called I call the new `update` method which invalidates the `lockKey` and `sessionHzKey` used so far so that they are recomputed on next usage of the respective getter method.

**Comment by jbee** [comment]
Jenkins test please

**Comment by jbee** [comment]
Jenkins test please

**Comment by jbee** [comment]
Jenkins test please

**Comment by jbee** [comment]
Jenkins test please

File: appserver/common/container-common/src/main/java/com/sun/enterprise/container/common/impl/util/ClusteredSingletonLookupImplBase.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (MattGill98) [reply appserver/common/container-common/src/main/java/com/sun/enterprise/container/common/impl/util/ClusteredSingletonLookupImplBase.java]: Ah I see! Could the method name be changed or a javadoc added to explain this?

File: appserver/tests/quicklook/ejb/remoteview/pom.xml [hunk 4]
```diff
@@ -87,6 +87,11 @@
                         <version>5.6</version>
                         <classifier>jdk15</classifier>
                     </dependency>
+					<dependency>
```
Comment (MattGill98) [inline appserver/tests/quicklook/ejb/remoteview/pom.xml]: Bleurgh tabs! Also, if this is a fix for `QA-171`, you'd be better making a separate PR for it, and then re-testing this PR.

Comment (jbee) [reply appserver/tests/quicklook/ejb/remoteview/pom.xml]: It was just an attempt. Worked locally on first try but sadly not here so I am going to remove it again.  Just wanted to test it out "quickly".

**Comment by jbee** [comment]
Jenkins test please

**Comment by jbee** [comment]
Jenkins test please

File: appserver/common/container-common/src/main/java/com/sun/enterprise/container/common/impl/util/ClusteredSingletonLookupImplBase.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (jbee) [reply appserver/common/container-common/src/main/java/com/sun/enterprise/container/common/impl/util/ClusteredSingletonLookupImplBase.java]: Changed it to `invalidateKeys` and added some javadoc too.

**Comment by jbee** [comment]
Jenkins test please
